src/components/primary-button/style.tsx

```diff
@@ -0,0 +1,25 @@
+import styled from 'styled-components/native';
+
+export const LoginText = styled.Text`
+  font-size: 16px;
+  font-weight: regular;
+  color: white;
+  font-weight: bold;
+`;
+
+export const LogginButton = styled.TouchableOpacity<{ disabled: boolean }>`
+  align-items: center;
+  justify-content: center;
+  width: 90%;
+  height: 44px;
+  background-color: ${props => (props.disabled ? '#7c65f4' : '#6c50f2')};
+  margin: auto;
+  padding: 10px;
+  border-radius: 15px;
+  margin-top: 20px;
+`;
+
+export const ActivityIndicatorButton = styled.ActivityIndicator`
```

Comment:
acho que esse nome deixa a entender que o ActivityIndicator é um Button, mas no caso seria o ActivityIndicator do Button, né? acho que ficaria mais claro trocando a ordem
```suggestion
export const ButtonActivityIndicator = styled.ActivityIndicator`
```